app/src/main/java/com/sunlightlabs/congress/services/LegislatorService.java

```diff
@@ -45,16 +45,148 @@
 		params.put("longitude", String.valueOf(longitude));
 		return legislatorsFor(Congress.url("legislators/locate", basicFields, params));
 	}
-	
+
+	public static List<Legislator> allForState(String state) throws CongressException {
+        // /members/{chamber}/{state}/current.json
+        String[] senate = new String[] { "members", "senate", state, "current" };
+        String[] house = new String[] { "members", "house", state, "current" };
+
+        List<Legislator> members = new ArrayList<Legislator>();
+
+        List<Legislator> senators = proPublicaLegislatorsFor(ProPublica.url(senate));
+        for (int i=0; i<senators.size(); i++) {
+            senators.get(i).state = state;
+            senators.get(i).chamber = "senate";
+        }
+
+        List<Legislator> representatives = proPublicaLegislatorsFor(ProPublica.url(house));
+        for (int i=0; i<representatives.size(); i++) {
+            representatives.get(i).state = state;
+            representatives.get(i).chamber = "house";
+        }
+
+        members.addAll(senators);
+        members.addAll(representatives);
+
+        return members;
```

Comment:
This function makes 2 API requests, one for each chamber, and then appends them to each other client-side to produce a unified client list of all senators and representatives for a given state/district.